### PR TITLE
Splunk postman collection

### PR DESCRIPTION
**Pull request by r-walker**
Per group decision, created "docs" directory. Added postman directory and moved the postman collections there.

### CODE REVIEW

**Review by zdc217 — APPROVED** [review]

**Review by ddriddle — APPROVED** [review]

File: docs/postman/README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ https://learning.postman.com/docs/postman/launching-postman/installation-and-upd
 1. Open Postman and choose import
     ![image](https://user-images.githubusercontent.com/4566388/82709416-2aec6280-9c46-11ea-8fae-f197ca728b0f.png)
 
-1. Drag and drop (or click Upload Files and browse to) the /test/postman/ClearPass API.postman_collection.json file in the repo. 
+1. Drag and drop (or click Upload Files and browse to) the /docs/postman/ClearPass API.postman_collection.json file (for example) in the repo.
```
Comment (MKletz) [inline docs/postman/README.md:38]: is this file path correct?

Comment (MKletz) [reply docs/postman/README.md:38]: it looks like that clearpass file is still in /test/

Comment (zdc217) [reply docs/postman/README.md:38]: I think it's correct. This reads to me like Ryan renamed the test folder to docs.

Comment (r-walker) [reply docs/postman/README.md:38]: I wasn't sure if you'd still want a test dir so I left that, but functionally it was like renaming test to docs

**Pull request merged**